src/Phinx/Db/Adapter/MysqlAdapter.php

```diff
@@ -414,15 +414,19 @@ public function addColumn(Table $table, Column $column)
      */
     public function renameColumn($tableName, $columnName, $newColumnName)
     {
-        $rows = $this->fetchAll(sprintf('DESCRIBE %s', $this->quoteTableName($tableName)));
+
+        $this->startCommandTimer();
```

Comment:
Why do you need the timer?

Reply:
Timer wasn't added by me...I think I need to update my branch with latest master. 

Tried resolving conflicts on github itself but I don't think thats helping.